cds/modules/flows/api.py

```diff
@@ -256,11 +258,10 @@ def build_steps(self):
             task_name='file_video_extract_frames'
         )
         parallel_tasks_group.append(video_extract_task)
-        for preset_quality in all_distinct_qualities:
-            transcode_task = self.create_task(task_name='file_transcode',
-                                              preset_quality=preset_quality,
-                                              )
-            parallel_tasks_group.append(transcode_task)
+        transcode_task = self.create_task(
+            flow=self, task_name='file_transcode', create_task_table=False
```

Comment:
We need this because we are using the same methods to manage all different tasks we have (ExtractMetadata, ExtractFrames and TranscodeVideo), until now we were creating one db `Task` per celery task, but now we don't want to create the task table in the db when creating the TranscodeVideo task. I don't like this parameter either but I cannot find a better solution, I am open to any suggestion!